scene/main/viewport.cpp

```diff
@@ -4252,7 +4252,7 @@ void Viewport::_camera_3d_remove(Camera3D *p_camera) {
 	camera_3d_set.erase(p_camera);
 	if (camera_3d == p_camera) {
 		camera_3d->notification(Camera3D::NOTIFICATION_LOST_CURRENT);
```

Comment:
`_camera_3d_set(nullptr);` also triggers such notification, remove this one to notify only once.
```suggestion
```

Reply:
Thank you for reviewing. Updated.